package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@edx/edx-proctoring",
   "//": "Be sure to update the version number in edx_proctoring/__init__.py",
   "//": "Note that the version format is slightly different than that of the Python version when using prereleases.",
-  "version": "2.1.9",
+  "version": "2.1.10",
```

Comment:
**Reviewer Note:** Temporary mismatch to manually test before reverting this line.